### PR TITLE
[FEATURE] Create new props LinearBar

### PR DESCRIPTION
**Pull request by viniieng**
### Description

Create new props Linear Bar

[BSWA-1911](https://adapcon.atlassian.net/jira/software/c/projects/BSWA/boards/52?assignee=712020%3A48dece61-7842-486e-8ecb-88d26bc648c6&selectedIssue=BSWA-1911)

### CODE REVIEW

File: package-lock.json
Comment (brunaboeger) [inline package-lock.json:1]: altere a versão do package-lock.json e package.json para "version": "4.3.12".

**Review by brunaboeger — CHANGES_REQUESTED** [review]

File: .nvmrc
Comment (gabrielforster) [inline .nvmrc:1]: isso aq nao deveria estar aq

**Review by gabrielforster — CHANGES_REQUESTED** [review]

File: src/components/DataVisualization/LinearBar/LinearBar.stories.mdx [hunk 1]
```diff
@@ -31,6 +31,18 @@ import { colors } from '@pb/utils/constants';
         },
         defaultValue: 'R$ 00.00',
       },
+      positionRight: {
```
Comment (gabrielforster) [inline src/components/DataVisualization/LinearBar/LinearBar.stories.mdx:34]: position right?

**Review by gabrielforster — CHANGES_REQUESTED** [review]

**Review by VictorTavaress — DISMISSED** [review]
LGTM

File: src/components/DataVisualization/LinearBar/LinearBar.vue [hunk 2]
```diff
@@ -1,6 +1,12 @@
 <template>
-  <div class="linear-progress-bar">
-    <div class="progress">
+  <div
+    class="linear-progress-bar"
+    :class="{ 'horizontal-value': horizontalValue }"
+  >
+    <div
+      class="progress"
+      :style="`width: ${barWidth}%; margin-bottom: 0;`"
```
Comment (CauaKath) [inline src/components/DataVisualization/LinearBar/LinearBar.vue:8]: wouldn't it be better if u pass a second paramenter called width type to allow the consumer to choose the width type between pixels (px), percent (%) or viewport width (vw)?

**Review by CauaKath — CHANGES_REQUESTED** [review]

File: src/components/DataVisualization/LinearBar/LinearBar.stories.mdx [hunk 3]
```diff
@@ -44,6 +56,8 @@ export const Template = (args, { argTypes}) => ({
         :label="label"
         :percent="percent"
         :value="value"
+        :horizontal-value=" horizontalValue"
```
Comment (brunaboeger) [inline src/components/DataVisualization/LinearBar/LinearBar.stories.mdx:59]:
```suggestion
        :horizontal-value="horizontalValue"
```

**Review by CauaKath — APPROVED** [review]
LGTM

**Review by VictorTavaress — APPROVED** [review]
LGTM

**Review by brunaboeger — APPROVED** [review]
LGTM

**Review by CauaKath — APPROVED** [review]
LGTM

**Review by gabrielforster — APPROVED** [review]
LGTM

**Pull request merged**